ci/run_ctests_cpu.sh

```diff
@@ -14,4 +14,4 @@ fi
 # LEGATE_TEST=1 to test broadcasting code paths (locally).
 LEGATE_CONFIG=${LEGATE_CONFIG:- --cpus 8 --gpus 0 --sysmem=6000 --omps=0} \
 LEGATE_TEST=${LEGATE_TEST:-1} \
-legate ./cpp_tests --output-on-failure --no-tests=error "$@" --gtest_filter="*CSV*"
+legate ./cpp_tests --output-on-failure --no-tests=error "$@" --gtest_filter="*CSV*:*Binary*"
```

Comment:
I don't know what coolaid github is drinking here to include this diff from the previous PR (happened yesterday as well).

Reply:
Nvm, something was wrong with the rebase, the last merge commit was there, but it was incorrect.